context/app/static/js/components/detailPage/ProcessedData/ProcessedDataWorkspaceMenu.tsx

```diff
@@ -56,7 +55,7 @@ function ProcessedDataWorkspaceMenu({
     initialSelectedDatasets: [uuid].filter(Boolean),
   });
 
-  const openEditWorkspaceDialog = useOpenDialog('ADD_DATASETS_FROM_SEARCH');
+  const openEditWorkspaceDialog = useOpenDialog('ADD_DATASETS_FROM_SEARCH', uuid);
```

Comment:
I'm a little confused as to why we need to pass uuid here, also to the `SelectableTableProvider` in this component and include a `SelectableTableProvider` without the `uuid` in the dataset component.